_source/_posts/2023-01-04-signin-custom-domain.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: blog_post
+title: "A Secure and Themed Sign-in Page"
+author: alisa-duncan
+by: advocate
+communities: [security,.net,java,javascript,go,php,python,ruby]
+description: "Redirecting to the Okta-hosted sign-in page is the most secure way to authenticate users in your application. But the default configuration yield a very neutral sign-in page. This post walks you through customization options and setting up a custom domain so the personality of your site shines all through the user's experience."
+tags: [security, theming]
+tweets:
+- "Add some flair to your sign-in process by following the quick steps in this post!"
+image: blog/signin-custom-domain/social.jpg
+type: awareness
+---
+
+Creating secure applications requires authentication. Delegating all the tedious details of the sign-in process to Okta is the most secure method to authenticate, not to mention speedier for development. So you'll see us advocating for and using the Okta-hosted sign-in page in our blog posts.
+
+But the default sign-in page can look too different from the personality you have in your application. And the login URL redirects to a domain outside your application, which is also different from what we want. Fortunately, you can add some flair to your Okta-hosted sign-in page, allowing your application's character to carry through in the user authentication process while maintaining the highest level of security. In other words, winning with the best of both worlds.
+
+![Giphy of Awkafina holding a trophy with confetti falling around her](https://media.giphy.com/media/26u4cqiYI30juCOGY/giphy.gif)
```

Comment:
nit: center image, not a big deal though, so don't worry about it too much, but _i think_ something like this _should_ work:
```suggestion
![Giphy of Awkafina holding a trophy with confetti falling around her](https://media.giphy.com/media/26u4cqiYI30juCOGY/giphy.gif){: .center-image}
```

Reply:
Yup! This works.